sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/AsyncBenchmark.java

```diff
@@ -47,7 +46,7 @@
 
 abstract class AsyncBenchmark<T> {
     private final MetricRegistry metricsRegistry = new MetricRegistry();
-    private final ScheduledReporter reporter;
+    public ScheduledReporter reporter;
```

Comment:
does this need to be public? can we change it to private or package level?

Reply:
reverted